tools/model_partition_tool/Graph.py

```diff
@@ -0,0 +1,441 @@
+#! /usr/bin/python
```

Comment:
It would be nice to specify python version if it depends on version.

Reply:
There is a strong dependency on python3 as the tflite interpreter functions to get operation details are supported only via python3 - tensorflow 2.x combination, as far as I recall.

Reply:
I have mentioned the dependency on python3 under README.md. Please execute the scripts as `python3 <test_script.py>`